src/components/products/Table.vue

```diff
@@ -66,6 +69,9 @@
           this.loading = false;
         }
       },
+      openProductPage(id) {
+        this.$router.push("/settings/product/" + id);
```

Comment:
template string gebruiken wanneer mogelijk
```suggestion
        this.$router.push(`/settings/product/${id}`);
```